lib/project_types/extension/models/specification_handlers/default.rb

```diff
@@ -19,7 +19,7 @@ def graphql_identifier
         end
 
         def name
-          message("name")
+          specification.name
```

Comment:
Are we sure this is the correct behaviour?  I think we might want to default to message("name") in case there was a local override but fallback to this:

`message("name") || specification.name`?

Reply:
Implemented as per your suggestion and added tests.